src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseEntitySelectPage.tsx

```diff
@@ -18,6 +18,29 @@ import CONST from '@src/CONST';
 import ROUTES from '@src/ROUTES';
 import type {QBOReimbursableExportAccountType} from '@src/types/onyx/Policy';
 
+function Footer({isTaxEnabled, isLocationsEnabled}: {isTaxEnabled: boolean; isLocationsEnabled: boolean}) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    if (isTaxEnabled) {
+        return (
+            <>
+                isTaxesEnabled && <Text style={[styles.mutedNormalTextLabel, styles.pt2]}>{translate('workspace.qbo.outOfPocketTaxEnabledDescription')}</Text>
+            </>
+        );
+    }
+
+    if (isLocationsEnabled) {
+        return (
+            <View style={[styles.flex1, styles.flexRow, styles.alignItemsCenter, styles.gap2, styles.mt1]}>
+                <Text style={styles.mutedTextLabel}>{translate('workspace.qbo.locationsAdditionalDescription')}</Text>
```

Comment:
Since we are displaying the `hint` text for `locations` here, can we not remove the display of the hint text within configurations page [here](https://github.com/Expensify/App/blob/e8ae3c5acedf0e6788dc574c7b6f3043ca37092a/src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseConfigurationPage.tsx#L56)?

Reply:
https://github.com/Expensify/App/pull/41638/commits/023c31358725cb9aab181bc076d7d05e05cdbb61